designs/administrator/future-openshift/bulk-actions/index.md

```diff
@@ -0,0 +1,139 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Bulk actions
+
+Bulk actions will be added to a number of list views in the OpenShift console. This feature will allow users to perform an action on multiple resources at once. The list of actions include adding labels, adding annotations and deleting resources.
+
+## Bulk actions in toolbar
+The bulk selector will be the first component in the toolbar on pages where bulk actions are available.
+![toolbar](img/toolbar.png)
+
+Users can click the checkbox in the bulk selector to select all items in the list. If all items are selected, the bulk selector checkbox will be filled with a check and the bulk selector will read ‘[#] selected’, informing users that all known items in the list are selected. Clicking the bulk selector checkbox again will deselect all items.
+![bulk-selector-states](img/bulk-selector-states.png)
```

Comment:
Just to make sure I understand, they can click either the checkbox to select all/deselect all, or just like on the "3 selected" to open the dropdown and choose that way?

Reply:
@itsptk correct!